### DISCUSSION TITLE
Botball wombat not connecting to create

### DISCUSSION THREAD

**Discussion by dogman1214** [iRobotEducation/create3_docs#560] [Questions] [answered]
### How are you connecting to your Create 3?

USB Ethernet (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Macbook Pro Sonoma 14.3

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

H.2.6

### Which RMW is your robot running?

FastRTPS

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

Yes

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

The KIPR institute Wombat controller keeps displaying the message "waiting for the create to connect". this has not happened with the previous create 2; what happened, and how can we fix this?
We have already tried re-updating the firmware (humble) multiple times already, and re-updating the Wombat controller.

**Comment by slowrunner** [comment]
@dogman1214 Disclaimer: I am just a Create3 user, not affiliated with iRobotEducation.

What version of the Wombat-OS are you running?  (30.2.5 supports Create3) 
What is the Create 3 App Configuration settings:
- RMW_IMPLEMENTATION:  rmw_fastrtps_cpp
- Enable Fast DDS discovery server?  checked?  
- Address and port of Fast DDS discovery server:   (did you fill it in?   You have to type it in even though it looks like it is there already)

Suggest download the Create3 log file to your laptop and then post it here.

Any chance you could post the link to the KIPR pdf instruction file for connecting your Controller to the Create3 and configuring everything?

The networking configuration in the OS files is a bit convoluted.  It looks like the discovery server is running on the Docker container IP, and the Raspberry Pi/Wombat Controller is at the expected 196.168.186.3.  Have you connected a laptop to the controller so you can verify the network interfaces are alive at the expected IP?

**Comment by slowrunner** [comment]
@dogman1214 It appears information about connecting and setting up the Create3 with the Wombat Controller must stay within the KIPR organized guidelines.  Please address your questions to your team instructor.   I don't know if iRobotEducation will be able to help you, I will not be able to.

Statement from KIPR:
```
All of our educators and students are provided with step-by-step instructions 
on how to connect the Wombat to the Create 3 through their team instructor. 
We keep these instructions behind a paid wall that is provided after each team registration.
```

**Comment by erinharrington-12** [comment, marked as answer]
@dogman1214 KIPR Software Developer here.

Please join our Botball Discord for any questions regarding the Create 3 connecting to the Wombat controller, located at kipr.org. Teams are provided step-by-step instructions on how to connect the Wombat to the Create 3. Please check your team's homebase for more details.

**Comment by dogman1214** [comment]
sorry, that was my friend using the account.
if you want his information,  ask me. (discord name...)
